Source/Core/DolphinQt2/MainWindow.cpp

```diff
@@ -844,9 +856,9 @@ void MainWindow::HideRenderWidget(bool reinit)
     setWindowTitle(QString::fromStdString(Common::scm_rev_str));
   }
 
-  // The following code works around a driver bug that would lead to Dolphin crashing when changing
-  // graphics backends (e.g. OpenGL to Vulkan). To avoid this the render widget is (safely)
-  // recreated
```

Comment:
Unrelated change

Reply:
the linter forces this change on me though :/